kratos/tests/test_model_part.py

```diff
@@ -237,6 +237,7 @@ def test_model_part_properties(self):
         self.assertEqual(model_part.HasProperties(1), True)
         random_sub_model_part = model_part.CreateSubModelPart("RandomSubModelPart")
         self.assertEqual(random_sub_model_part.HasProperties(1), False)
+        self.assertEqual(random_sub_model_part.RecursivelyHasProperties(1), True)

```

Comment:
This is a test @philbucher

Reply:
ah ok didn't see

Reply:
Ninja test

![imagen](https://user-images.githubusercontent.com/19991680/51794167-29836500-21cd-11e9-9c6e-d23121d19328.png)